### DISCUSSION TITLE
Getting Forbidden Error's, with the permissions mentioned in the docs.

### DISCUSSION THREAD

**Discussion by Haptein** [Pycord-Development/pycord#2357] [Q&A] [answered]
I have a bot in a few servers (not mine) and I'm getting forbidden errors when the bot has the needed permissions.

Case 1: I'm trying to create a private channel, the bot has manage roles and manage channels permission.
```
# category_channel: Optional[discord.CategoryChannel]
# member: discord.Member - corresponds to a member just joining the server.

overwrites = {
        guild.default_role: discord.PermissionOverwrite(read_messages=False),
        member: discord.PermissionOverwrite(read_messages=True, send_messages=True),
        guild.me: discord.PermissionOverwrite(read_messages=True, send_messages=True, manage_messages=True)
    }

    channel = await guild.create_text_channel(name, overwrites=overwrites, category=category_channel)
```

Case 2: I'm trying to get the guild.invites, with the bot having manage server permission but I get missing permissions.

I'm very confused as to why this is happening, any guidance or help would be super appreciated 🙏

**Comment by Lulalaby** [comment, marked as answer]
Please use our discord server for such types of questions

**Reply by Haptein** [reply]
Got it!
